packages/tasit-account/src/ethers.test.js

```diff
@@ -120,61 +120,64 @@ describe("ethers.js - unit tests", function() {
     address: "0x6fC21092DA55B392b045eD78F4732bff3C580e2c",
   };
 
-  it("should resolve ENS name", async function() {
+  xit("should resolve ENS name", async function() {
     const address = await ensSample.provider.resolveName(ensSample.name);
     expect(address).to.equal(ensSample.address);
   });
 
-  it("should lookup ENS address", async function() {
+  xit("should lookup ENS address", async function() {
     const name = await ensSample.provider.lookupAddress(ensSample.address);
     expect(name).to.equal(ensSample.name);
   });
 
   it("should instantiate a contract object", async function() {
     const contractABI = [
       "event ValueChanged(address indexed author, string oldValue, string newValue)",
-      "constructor(string value)",
-      "function getValue() view returns (string value)",
-      "function setValue(string value)",
+      "constructor(string memory) public",
```

Comment:
Out of curiosity, what motivated the `memory` addition? The change to `public` makes sense for testing.

Reply:
It's a solc 0.5 requirement (I haven't the chance to read about that, but the contract wasn't compiled without it). I've just realized that I can change solidity version to use 0.4.x (truffle.conf) and then I'll came back to original contract source code. Is it make sense or you want to use 0.5 solc?

Reply:
Hmm. We might as well move over to 0.5 now to future-proof things a bit. But that context is really helpful.

Down the road I think we'll want to use whichever solc setting the contracts under test are using.

1) The test contracts that we include for testing ethers at a low level should always be the latest solc version supported by ethers.

2) When we use contracts from other projects and run their existing test suites + our own new tests against those contracts using `tasit-act`, we'll want to stick with the current settings (solc version, Solidity version, Truffle version, etc.) of that project for that part of the test suite.

Reply:
So regarding this section in this PR, I think it can stay as-is. Do you agree?